src/main/java/com/iota/iri/model/persistables/Approvee.java

```diff
@@ -3,13 +3,25 @@
 import com.iota.iri.model.Hash;
 
 /**
- * Created by paul on 5/15/17.
+ * This is a collection of hash identifiers that can change the values of an <tt>Approvee</tt>
```

Comment:
what do you mean by "can change the values"?

Reply:
I think it's c/p from address :)